[2023-07-13 21:10:12,539] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: ETL_stock_data_google_DAG.extract manual__2023-07-13T21:10:11.387326+00:00 [queued]>
[2023-07-13 21:10:12,548] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: ETL_stock_data_google_DAG.extract manual__2023-07-13T21:10:11.387326+00:00 [queued]>
[2023-07-13 21:10:12,549] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 21:10:12,550] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2023-07-13 21:10:12,551] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 21:10:12,563] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): extract> on 2023-07-13 21:10:11.387326+00:00
[2023-07-13 21:10:12,568] {standard_task_runner.py:52} INFO - Started process 285 to run task
[2023-07-13 21:10:12,571] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_stock_data_google_DAG', 'extract', 'manual__2023-07-13T21:10:11.387326+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/ETL_stock_data_google_DAG.py', '--cfg-path', '/tmp/tmpsp4tx4kq', '--error-file', '/tmp/tmp4lm4_o69']
[2023-07-13 21:10:12,573] {standard_task_runner.py:80} INFO - Job 25: Subtask extract
[2023-07-13 21:10:12,635] {logging_mixin.py:109} INFO - Running <TaskInstance: ETL_stock_data_google_DAG.extract manual__2023-07-13T21:10:11.387326+00:00 [running]> on host a917c5d777dc
[2023-07-13 21:10:12,686] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=fvinciguerra
AIRFLOW_CTX_DAG_ID=ETL_stock_data_google_DAG
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2023-07-13T21:10:11.387326+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-13T21:10:11.387326+00:00
[2023-07-13 21:10:13,049] {python.py:175} INFO - Done. Returned value was:                          Date        Open  ...  Stock Splits  Stock_Splits
0   2022-07-14 00:00:00-04:00  110.825996  ...           0.0             0
1   2022-07-15 00:00:00-04:00  112.962997  ...           0.0             0
2   2022-07-18 00:00:00-04:00  113.440002  ...          20.0             0
3   2022-07-19 00:00:00-04:00  111.730003  ...           0.0             0
4   2022-07-20 00:00:00-04:00  114.059998  ...           0.0             0
..                        ...         ...  ...           ...           ...
246 2023-07-07 00:00:00-04:00  120.889999  ...           0.0             0
247 2023-07-10 00:00:00-04:00  119.070000  ...           0.0             0
248 2023-07-11 00:00:00-04:00  116.760002  ...           0.0             0
249 2023-07-12 00:00:00-04:00  119.300003  ...           0.0             0
250 2023-07-13 00:00:00-04:00  121.540001  ...           0.0             0

[251 rows x 9 columns]
[2023-07-13 21:10:13,074] {xcom.py:447} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config.
[2023-07-13 21:10:13,075] {taskinstance.py:1743} WARNING - We expected to get frame set in local storage but it was not. Please report this as an issue with full logs at https://github.com/apache/***/issues/new
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1340, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1477, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1544, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2201, in xcom_push
    XCom.set(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 67, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/xcom.py", line 155, in set
    value=cls.serialize_value(value),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/xcom.py", line 445, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/local/lib/python3.8/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.8/site-packages/frozendict/__init__.py", line 32, in default
    return BaseJsonEncoder.default(self, obj)
  File "/usr/local/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1741, in get_truncated_error_traceback
    execution_frame = _TASK_EXECUTION_FRAME_LOCAL_STORAGE.frame
AttributeError: '_thread._local' object has no attribute 'frame'
[2023-07-13 21:10:13,077] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1340, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1477, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1544, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2201, in xcom_push
    XCom.set(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 67, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/xcom.py", line 155, in set
    value=cls.serialize_value(value),
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/xcom.py", line 445, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/local/lib/python3.8/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.8/site-packages/frozendict/__init__.py", line 32, in default
    return BaseJsonEncoder.default(self, obj)
  File "/usr/local/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2023-07-13 21:10:13,087] {taskinstance.py:1278} INFO - Marking task as FAILED. dag_id=ETL_stock_data_google_DAG, task_id=extract, execution_date=20230713T211011, start_date=20230713T211012, end_date=20230713T211013
[2023-07-13 21:10:13,099] {standard_task_runner.py:93} ERROR - Failed to execute job 25 for task extract (Object of type DataFrame is not JSON serializable; 285)
[2023-07-13 21:10:13,108] {local_task_job.py:154} INFO - Task exited with return code 1
[2023-07-13 21:10:13,137] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
